alfresco-transform-tika/alfresco-transform-tika-boot/src/main/resources/licence/3rd-party/Perl-GNU-license.txt

```diff
@@ -0,0 +1,248 @@
+		    GNU GENERAL PUBLIC LICENSE

```

Comment:
As per previous comment - remove - we're using via Perl Artistic license.

Reply:
GNU licences removed.